### PR TITLE
Add require to ensure Time#advance works without implicit required

### PR DESCRIPTION
**Pull request by qnm**
### The Issue

Using `Time#advance` without first loading `active_support/core_ext/date/calculations` (like Padrino may do) will result in an exception being raised.

I've created a testable code snippet at https://gist.github.com/qnm/63799aeb77cb6b9157f7
### What I expect to happen

I expect the time to be advanced according to the supplied arguments
### What actually happens

An exception is raised

### CODE REVIEW

File: activesupport/.ruby-version [hunk 1]
```diff
@@ -0,0 +1 @@
+2.1.4
```
Comment (arunagw) [inline activesupport/.ruby-version]: Seems like this file added in git commit. Can you remove this ?

Comment (qnm) [inline activesupport/.ruby-version]: Oops, apologies.

**Comment by qnm** [comment]
@arunagw Anything I can do to move this along?

**Comment by arunagw** [comment]
cc/ @senny

**Comment by senny** [comment]
@qnm @arunagw thanks guys :yellow_heart:

**Pull request merged**